packages/clerk-js/src/core/resources/Passkey.ts

```diff
@@ -43,13 +50,6 @@ export class Passkey extends BaseResource implements PasskeyResource {
     }).then(res => new Passkey(res?.response as PasskeyJSON));
   }
 
-  /**
-   * TODO-PASSKEYS: Implement this later
-   *
-   * GET /v1/me/passkeys
-   */
-  static async get() {}
-
```

Comment:
Instead we grab the passkeys directly from the User resource without performing a http request.